UserGuide.md

```diff
@@ -535,7 +549,22 @@ The optional `--delete_failed `flag will overwrite an existing failed run_report
 
 `REPORT` is the ID or name of the report you wish to generate for the run.  If you just completed step 2 above and created a report, the ID of that report should have been included in the results from that command.  If not, you can find the report using `carrot_cli report find`.
 
-Once the report has been generated, it will be available in 3 forms: the Jupyter notebook that is produced by CARROT before being run, the filled Jupyter notebook that is created once all the cells are run, and a static HTML representation of the filled notebook.
+
+##### <a name="manually-generating-a-report-for-several-runs"/> **Manually Generating a Report for Several Runs**
+
+The third option allows you to generate a report from the results of a query for runs.  Similar to the way it is possible to use carrot_cli to query for a list of runs matching certain parameters using the commands `carrot_cli pipeline find_runs`, `carrot_cli template find_runs`, and `carrot_cli test find_runs`, it is also possible to query for runs and have them used to generate a report using the commands `carrot_cli pipeline create_report_for_runs`, `carrot_cli template create_report_for_runs`, and `carrot_cli test create_report_for_runs`.
+
+These commands all have the format:
+
+```
+carrot_cli pipeline create_report_for_runs [options] PIPELINE REPORT
+```
+
+`PIPELINE` is the ID or name of the pipeline to which the runs you wish to query belong.
+
+`REPORT` is the ID or name of the report you wish to generate for the run.  If you just completed step 2 above and created a report, the ID of that report should have been included in the results from that command.  If not, you can find the report using `carrot_cli report find`.
+
+`[options]` can be any from a list of options for filtering runs that are the same as you have available to you for any of the `find_runs` commands. For a full list of the filtering options available, see the output of `carrot_cli pipeline create_report_for_runs --help`.
```

Comment:
I would add more detail here about what options are likely to be sueful... If i don't provide any options at all do i just get the outputs of every run ever completed for PIPELINE by default?